### DISCUSSION TITLE
What is logically decentralized?

### DISCUSSION THREAD

**Discussion by jaybny** [bytemaster/clarion#9] [General]
https://twitter.com/bytemaster7/status/1366945977031135236

So, 

what is meant by "logically decentralized communication platform" 

is it anything like https://matrix.org/ ? 

thanks for the replies.

**Comment by Dexaran** [comment]
When people talk about [software] decentralization, there are actually three separate axes of centralization/decentralization that they may be talking about ([source](https://medium.com/@VitalikButerin/the-meaning-of-decentralization-a0c92b76a274)):

- Architectural decentralization 
- Political decentralization
- Logical decentralization

For better understanding of the 'logical decentralization' you can read this article: 
https://moreequalanimals.com/posts/logical-decentralization
